.github/ISSUE_TEMPLATE/10-bug-report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: File a bug report related to running the package
+title: "[Bug] "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please, before submitting, make sure that:
+
+        - There is not an [existing issue](https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/issues) with the same question
+        - You have read the [contributing guide](https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/blob/main/docs/src/90-contributing.md/)
```

Comment:
Should the link point to the documentation in the end?

Reply:
Will update the template.